### PR TITLE
G 67

### PR DESCRIPTION
**Pull request by Nexengineer**

### CODE REVIEW

File: src/In.ProjectEKA.HipService/DataFlow/DataFlowClient.cs [hunk 1]
```diff
@@ -33,19 +33,18 @@ public class DataFlowClient
             IEnumerable<Entry> data,
             KeyMaterial keyMaterial)
         {
-            var url = await centralRegistryClient.GetUrlFor(dataRequest.ConsentManagerId);
-            url.MatchSome(async providerUrl => await PostTo(providerUrl,
-                dataRequest.ConsentId,
+             await PostTo(dataRequest.ConsentId,
```
Comment (ganesanarun) [inline src/In.ProjectEKA.HipService/DataFlow/DataFlowClient.cs:36]: Codacy found an issue: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5745784)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/DataFlow/DataFlowNotificationRequest.cs  1
- src/In.ProjectEKA.HipService/Gateway/Model/GatewayDataNotificationRequest.cs  1
         

Clones removed
==============
+ src/In.ProjectEKA.HipLibrary/Patient/Model/DataRequest.cs  -2
+ src/In.ProjectEKA.HipService/DataFlow/DataRequest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5745784&bid=18799945)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/DataFlow/DataFlowNotificationRequest.cs  1
- src/In.ProjectEKA.HipService/Gateway/Model/GatewayDataNotificationRequest.cs  1
         

Clones removed
==============
+ src/In.ProjectEKA.HipLibrary/Patient/Model/DataRequest.cs  -2
+ src/In.ProjectEKA.HipService/DataFlow/DataRequest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5745784&bid=18799945)

File: src/In.ProjectEKA.HipService/Common/Model/ConsentArtefact.cs [hunk 2]
```diff
@@ -16,6 +16,10 @@ public class ConsentArtefact
         public ConsentPermission Permission { get; }
         public IEnumerable<GrantedContext> CareContexts { get; }
 
+        public ConsentArtefact()
+        {
+        }
```
Comment (dheeraj-p) [inline src/In.ProjectEKA.HipService/Common/Model/ConsentArtefact.cs:21]: Any need for this empty constructor?

Comment (Nexengineer) [reply src/In.ProjectEKA.HipService/Common/Model/ConsentArtefact.cs:21]: This for generating faker object

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/Gateway/Model/GatewayDataNotificationRequest.cs  1
- src/In.ProjectEKA.HipService/DataFlow/DataFlowNotificationRequest.cs  1
         

Clones removed
==============
+ src/In.ProjectEKA.HipService/DataFlow/DataRequest.cs  -2
+ src/In.ProjectEKA.HipLibrary/Patient/Model/DataRequest.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5745784&bid=18799945)

**Pull request merged**